hypothesis-python/tests/nocover/test_from_type_recipe.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import, division, print_function
+
+import datetime
+import decimal
+import fractions
+import numbers
+import uuid
+
+import hypothesis.strategies as st
+from hypothesis import given
+
+
+def everything_except(excluded_types):
+    """Recipe copied from the docstring of ``from_type``"""
+    return (
+        st.from_type(type)
+        .flatmap(lambda x: st.from_type(x))
+        .filter(lambda x: not isinstance(x, excluded_types))
+    )
+
+
+many_types = [
+    bool,
+    bytearray,
+    bytes,
```

Comment:
Use `hypothesis.internal.compat.binary_type`, and `text_type` instead of `str` below.  Or just delete `bytes` from this list, that would be easier :smile: